netket/abstract_variational_driver.py

```diff
@@ -46,15 +46,15 @@ def _to_iterable(maybe_iterable):
 class AbstractVariationalDriver(abc.ABC):
```

Comment:
Looks like this fils was moved to `netket/drivers/abstract_variational_driver.py`, so should be removed here?

Reply:
thanks. Yeah, I did a mess rebasing.